### PR TITLE
V2 : Fix AMF Deserializer

### PR DESCRIPTION
**Pull request by z3dev**
UG.. Missed this one!

These changes correct translate() to provide V2 require and export statements.

### All Submissions:

* [X] Have you followed the guidelines in our Contributing document?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [X] Does your submission pass tests?

### CODE REVIEW

File: packages/io/amf-deserializer/index.js [hunk 1]
```diff
@@ -35,7 +36,7 @@ const instantiate = require('./deserialize')
 const deserialize = (input, filename, options) => {
   const defaults = {
     output: 'jscad',
-    version: '0.0.0',
+    version: version,
```
Comment (kaosat-dev) [inline packages/io/amf-deserializer/index.js]: in modern js, you do not need to repeat the keys, so this should be 
```
{ output: 'jscad',
 version,
addMetaData: true
  }
```

File: packages/io/amf-deserializer/index.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ History:
 //
 // //////////////////////////////////////////
 
+const version = require('./package.json').version
```
Comment (kaosat-dev) [inline packages/io/amf-deserializer/index.js]: hmm this is nice & simple but it changes the significance of the 'version'  compared to previous versions (ie jscad version versus deserializer version), I think it is ok though :)

Comment (kaosat-dev) [reply packages/io/amf-deserializer/index.js]: we should just make that clear

File: packages/io/amf-deserializer/index.js (hunk 1 above)
Comment (z3dev) [reply packages/io/amf-deserializer/index.js]: Understood. Let's catch these via lint. There are plenty.

File: packages/io/amf-deserializer/index.js (hunk 2 above)
Comment (z3dev) [reply packages/io/amf-deserializer/index.js]: good idea. i changed the metadata information about the producer, specifically saying JSCAD IO library. i'll update the other IO libraries as well.

**Pull request merged**